components/manager-ip-setter/config/cloudify-manager-ip-setter.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Cloudify Manager IP Setter
+Before=cloudify-mgmtworker.service cloudify-amqpinflux.service cloudify-riemann.service logstash.service
+After=network.target cloudify-restservice.service
+
+[Service]
```

Comment:
maybe should be a `Type=oneshot`? does systemd try to restart the script after it exits?